### DISCUSSION TITLE
Dynamic Routing

### DISCUSSION THREAD

**Discussion by ahadali0500** [vercel/next.js#55800] [App Router] [answered]
I have make a website using next js and it contain dynamic routing which is working perfecting on localhost but I have uploaded on godaddy and vercel on both routes work  but dynamic routes [id] like this not working on live server how to solve it. Is there any configrations for it??

**Comment by icyJoseph** [comment]
Are you using export mode? That one I think can't handle dynamic routes IIRC, let me find some sources:

- https://github.com/vercel/next.js/discussions/55393

**Reply by ahadali0500** [reply, marked as answer]
Its Strange! It mean dynamic routing is not available in next js on live mode so with out it its not possible to dynamic web applications.so why react is saying use next js???
